Tasks/ANT/anttask.ts

```diff
@@ -1,15 +1,18 @@
 /// <reference path="../../definitions/vsts-task-lib.d.ts" />
 
 import tl = require('vsts-task-lib/task');
+import path = require('path');
```

Comment:
some user inputs ex: classFilesDirectory will be available only when code coverage is selected.